src/main/js/component/ConfigButtons.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { submitContainers, submitButtons } from '../../css/main.css';
+
+export default class ConfigButtons extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const includeTest = this.props.includeTest || "false";
```

Comment:
In the near future we need to spend time getting Redux working with this application.  There is a mechanism to set the default property values.  We don't want to set them in the render method.  I would prefer const {includeTest, buttonText, buttonType, buttonTestText, buttonTestType} = this.props;